vision/object_detection_segmentation/ssd-mobilenetv1/README.md

```diff
@@ -0,0 +1,137 @@
+# SSD-MobilenetV1
+
+## Description
+
+SSD-MobilenetV1 is an object detection model that uses a Single Shot MultiBox Detector(SSD) approach to predict object classes for boundary boxes. 
+
+SSD enables the model to only need to take one single shot to detect multiple objects in an image, and mobilenet is a base network that provides high-level features for object detection. The combination of these two model frameworks produces an efficent, high-accuracy detection model that requires less computational powers.
```

Comment:
```suggestion
SSD enables the model to only need one single shot to detect multiple objects in an image, and MobileNet is a base network that provides high-level features for object detection. The combination of these two model frameworks produces an efficient, high-accuracy detection model that requires less computational cost.
```
Perhaps talk a bit about the model architecture here for SSD and MobileNet? CNN vs RNN etc.